### DISCUSSION TITLE
Week 12 Project

### DISCUSSION THREAD

**Discussion by kassyoveson** [USU-ITLS-x265-23-summer/class-repo#224] [General]
This week I added boarders, columns, menu and changes around colors and fonts to help the design flow better. I am not awesome at coding, so it is a very basic design. I am just happy I was able to make something! Let me know what suggestions you have. 

[USU-ITLS-x265-23-summer/Kassy-Oveson--Project (github.com)](https://github.com/USU-ITLS-x265-23-summer/Kassy-Oveson--Project)

**Comment by CJ3315** [comment]
Great additions! These will definitely improve the looks of the cite!

**Comment by audreyhki** [comment]
Thanks for sharing Kassy, we've come so far from where we were at the beginning of the semester!
